theseus/geometry/se3.py

```diff
@@ -253,7 +253,7 @@ def transform_from(
     ) -> Point3:
         self._transform_shape_check(point)
         p = point if isinstance(point, torch.Tensor) else point.tensor
-        return Point3(SE3_base.transform_from(self.tensor, p, jacobians=jacobians))
+        return Point3(SE3_base.transform(self.tensor, p, jacobians=jacobians))
```

Comment:
I am guessing tests are failing since this update hasn't touched the theseus unit tests?

Reply:
Oh I see, maybe it is because pypi for torchlie hasn't updated.

Should we make the CI for theseus use from-source-install of torchlie? At least until torchlie moves to a standalone repo.

Reply:
Yeah, I already fixed this in #550 on @fantaosha's suggestion.